ccd-definition/CaseEvent/CareSupervision/MultiState.json

```diff
@@ -720,5 +720,23 @@
     "CallBackURLAboutToStartEvent": "${CCD_DEF_CASE_SERVICE_BASE_URL}/callback/message-judge/about-to-start",
     "CallBackURLAboutToSubmitEvent": "${CCD_DEF_CASE_SERVICE_BASE_URL}/callback/message-judge/about-to-submit",
     "CallBackURLSubmittedEvent": "${CCD_DEF_CASE_SERVICE_BASE_URL}/callback/message-judge/submitted"
+  },
+  {
+    "LiveFrom": "01/01/2017",
+    "CaseTypeID": "CARE_SUPERVISION_EPO",
+    "ID": "removeManagingOrganisation",
+    "Name": "Remove managing organisation",
+    "Description": "Removes managing organisation",
+    "DisplayOrder": 14,
+    "PreConditionState(s)": "Submitted;Gatekeeping;PREPARE_FOR_HEARING;FINAL_HEARING",
+    "PostConditionState": "*",
+    "SecurityClassification": "Public",
+    "ShowSummary": "Y",
+    "ShowEventNotes": "N",
+    "EndButtonLabel": "Save and continue",
+    "CallBackURLAboutToStartEvent": "${CCD_DEF_CASE_SERVICE_BASE_URL}/callback/remove-managing-organisation/about-to-start",
+    "CallBackURLAboutToSubmitEvent": "${CCD_DEF_CASE_SERVICE_BASE_URL}/callback/remove-managing-organisation/about-to-submit",
+    "CallBackURLSubmittedEvent": "${CCD_DEF_CASE_SERVICE_BASE_URL}/callback/remove-managing-organisation/submitted",
+    "EventEnablingCondition": "outsourcingPolicy.Organisation.OrganisationID != \"\""
```

Comment:
is this field not subject to this bug https://tools.hmcts.net/jira/browse/CCD-1123?

Reply:
yes it is, but in this particular setup it works. If outsourcingPolicy does not exists, CCD evaluation fails giving false value to entire expression, which is what we want here. If outsourcingPolicy exists then it is evaluated as expected. 
The problem is if conditions is like:
```
outsourcingPolicy.Organisation.OrganisationID **==**  \"\""
```
then if outsourcingPolicy is missing, CCD due to a bug, evaluates it as false which is not expected in such scenario

Reply:
👍